ocs_ci/utility/utils.py

```diff
@@ -1303,3 +1304,39 @@ def wait_for_co(operator):
     from ocs_ci.ocs.ocp import OCP
     ocp = OCP(kind='ClusterOperator')
     ocp.get(operator)
+
+
+def censore_values(data_to_censore, keys):
+    """
+    This function censores values in dictionary for passed keys
```

Comment:
What do you think about some "automatic" process - hide any values for keys containing `passw` or `token`?

Reply:
I did change of this function @dahorak @vavuthu , please take a look if you like this approach more.